Microsoft.Toolkit.Uwp.UI.Controls.Input/ColorPicker/ColorPreviewer.xaml

```diff
@@ -0,0 +1,120 @@
+                              VerticalAlignment="Center"
+                              Visibility="{Binding RelativeSource={RelativeSource TemplatedParent}, Path=ShowAccentColors, Converter={StaticResource BoolToVisibility}}">
+                            <Grid.ColumnDefinitions>
+                                <ColumnDefinition Width="*" />
+                                <ColumnDefinition Width="*" />
+                            </Grid.ColumnDefinitions>
+                            <Border x:Name="CenterCheckeredBackgroundBorder"
+                                    Grid.Column="0"
+                                    Grid.ColumnSpan="2"
+                                    HorizontalAlignment="Stretch"
+                                    VerticalAlignment="Stretch" />
+                            <Border x:Name="P1PreviewBorder"
+                                    Grid.Column="0"
+                                    Tag="1">
+                                <Border.Background>
+                                    <SolidColorBrush Color="{Binding RelativeSource={RelativeSource TemplatedParent}, Path=HsvColor, Converter={StaticResource AccentColor}, ConverterParameter='1'}" />
+                                </Border.Background>
+                            </Border>
+                            <Border x:Name="P2PreviewBorder"
+                                    Grid.Column="1"
+                                    CornerRadius="0,4,4,0"
+                                    Tag="2">
+                                <Border.Background>
+                                    <SolidColorBrush Color="{Binding RelativeSource={RelativeSource TemplatedParent}, Path=HsvColor, Converter={StaticResource AccentColor}, ConverterParameter='2'}" />
+                                </Border.Background>
+                            </Border>
+                        </Grid>
+                        <Border x:Name="ShadowTargetBorder"
+                                Grid.Row="0"
+                                Grid.Column="0"
+                                Grid.ColumnSpan="3"
+                                HorizontalAlignment="Stretch"
+                                VerticalAlignment="Stretch" />
+                        <!-- Must be last for drop shadow Z-index -->
+                        <Grid Grid.Column="1">
+                            <Border Background="{ThemeResource FlyoutPresenterBackground}"
+                                    CornerRadius="4"
+                                    HorizontalAlignment="Stretch"
+                                    VerticalAlignment="Stretch" />
+                            <Border x:Name="RightCheckeredBackgroundBorder"
+                                    CornerRadius="4"
+                                    HorizontalAlignment="Stretch"
+                                    VerticalAlignment="Stretch" />
+                            <Border x:Name="PreviewBorder"
```

Comment:
This is a rounded rectangle element, `AttachedCardShadow` should just be used here. Note that you need to wrap it in another border (as in the sample) for that to work as it's a primitive. That should make this easier.

Reply:
> AttachedCardShadow should just be used here.

I believe that is in the Media package so I can't use it. That was what I tried first.

Reply:
🤦‍♂️ I realized thinking that last night and forgot when I got to commenting here today. 🤔 I'll have to pull down the branch and take a look. Is the current issue that it's not displaying correctly? I have some thoughts on what may be happening. FYI @Ryken100, I know you've been poking at some alternatives too.

Reply:
> Is the current issue that it's not displaying correctly? 

The shadow isn't displaying on the target at all. I think it can't even find the target border due to namescope issues but am not sure.